### PR TITLE
AST-2286 - fix: blog layout 2 & 3 post title & meta not disabling

### PR DESCRIPTION
**Pull request by neerajmasai**
… hiding

### Description

- Fix: blog archives layout 2 & 3 post title structure title & meta hide option not working.

### Screenshots

Bug - https://share.getcloudapp.com/P8u76qo9
Fix - https://d.pr/i/YiokAb

### How has this been tested?

Steps To Reproduce: 

Go to the customizer and set in the blog archive settings to Layout 2 or 3.

Uncheck the title in Post Structure.

Check the front end.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by yssambare12 — COMMENTED** [review]
@neerajmasai Update branch with base branch

File: inc/blog/blog.php [hunk 1]
```diff
@@ -535,3 +535,23 @@ function astra_banner_elements_order( $structure = array() ) {
 		}
 	}
 }
+
+/**
+ * Check whether blogs post structure title & meta is disabled or not
+ */
+if ( ! function_exists( 'astra_is_blog_title_meta_disabled' ) ) {
```
Comment (imnavanath) [inline inc/blog/blog.php:542]: Don't make functions pluggable, you can remove this condition.

**Comment by imnavanath** [comment]
@neerajmasai add the changelog & fix the possible fata error bug.

**Comment by neerajmasai** [comment]
Closing this as a newer PR solves this bug - https://github.com/brainstormforce/astra-addon/pull/1472

**Pull request closed without merging**